docs/manual/docs/administrator-guide/configuring-the-catalog/system-configuration.md

```diff
@@ -272,3 +272,8 @@ Allows to view metadata history
 ## Harvesting
 
 *Allow editing on harvested records*: Enables/Disables editing of harvested records in the catalogue. By default, harvested records cannot be edited.
+
+## Geonetwork-ui configuration
+
+*Enable datahub*:
+You can adapt your GeoNetwork UI by enabling Datahub. First, download the specific plugin datahub-integration. Then, on the settings page, enable Datahub . you can provide an other specific Datahub configuration than the default one. You can also configure it for each portal. Ensure that Datahub is enabled in the settings to make it work on portals.
```

Comment:
Thanks for documenting your work. IMO this should go in a separate section of the documentation. The things that need to be documented are:
* what the plugin does (with a link to GeoNetwork-UI project)
* how to build the plugin
* how to enable it
* how to configure the datahub instances (link to the relevant documentation in GN-UI)

We might start a new "plugins" section in the documentation for that, I think that'd be appropriate. @jodygarnett do you have any opinion on how/where to put this? thanks!